u2-routing-controllers/features/tempController.js

```diff
@@ -0,0 +1,26 @@
+// /:unit1-to-:unit2/:temp1
+
+const convertTemp = (req, res) => {
+  console.log(req.params)
+
+  const u1 = req.params.unit1.toUpperCase()
+  const u2 = req.params.unit2.toUpperCase()
+  const t1 = parseInt(req.params.temp1)
+  console.log(u1, u2, t1)
+  let t2
+  if (u1 === u2) {
+    t2 = t1
+  } else if (u1 === 'C') {
+    t2 = (t1 * 9) / 5 + 32
+  } else if (u1 === 'F') {
+    t2 = ((t1 - 32) * 5) / 9
+  }
```

Comment:
:information_source: Note: `t2` will be undefined if either `u1` or `u2` are anything except C/c/F/f. In a real app you would probably want to handle those cases.